templates/advantage/index.html

```diff
@@ -58,7 +58,7 @@ <h4>Your personal subscription</h4>
 
```

Comment:
–1: the detach command doesn’t work.

> `ubuntu-advantage: error: unrecognized arguments: {token was here}`

The correct command is just `ua detach` without anything following.

Reply:
Done, removed the token from detach.